doc/user-manual/getting-started/installation.rst

```diff
@@ -22,7 +22,42 @@ In case of installation problems, check the section on :ref:`troubleshooting <tr
 Step 1: Install Agda
 ====================
 
-There are at least three options for installing Agda:
+There are at least four options for installing Agda:
+
+.. _prebuilt-agda-from-github:
+
+Option 0: Use Pre-Built Agda Binaries (experimental)
+-----------------------------------------------------
+
+Pre-built binaries are available for the following platforms:
+
+* Windows (x86-64)
+* Linux (x86-64, statically linked)
+* macOS (x86-64)
+* macOS (arm64, i.e. Apple M-series)
+
+They can be downloaded from the `Agda GitHub releases <https://github.com/agda/agda/releases/>`_ page (under "Assets").
+
+To install, download the appropriate archive, extract the binary, and place it in a directory listed in your ``PATH`` environment variable.
+
+.. important::
+   This option does not require GHC. However, Agda programs cannot be compiled
+   into executables without it. If you need to compile Agda programs, follow the
+   `GHCup installation instructions <https://www.haskell.org/ghcup/>`_ to install GHC.
+
+.. note::
+   These binaries are compiled with the :option:`optimise-heavily` flag only,
```

Comment:
Would be good to add that `enable-cluster-counting` is off in these prebuilt binaries.